src/pudl/io_managers.py

```diff
@@ -726,6 +770,13 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
                 },
             ).assign(sched_table_name=sched_table_name)
 
+        primary_key = self._get_primary_key(table_name)
+        context_cols = [
+            col for col in primary_key if col not in {"filing_name", "publication_time"}
+        ]
+        deduped = self.use_latest_filing_for_context(df, context_cols)
```

Comment:
Just for readability / ease of mental modeling I find named keyword arguments easier, or using the same variable names where it's arbitrary -- e.g. maybe rename `context_cols` to `unique_cols` here, or:

```py
        deduped = self.use_latest_filing_for_context(df, unique_cols=context_cols)
```